src/utils/index.ts

```diff
@@ -477,10 +479,47 @@ export function authorizationDataToAuthorization(auth: AuthorizationData): Autho
 /**
  * @hidden
  */
-export function numberToBalance(value: number | BigNumber, context: Context): Balance {
+export function numberToBalance(
+  value: number | BigNumber,
+  context: Context,
+  divisible?: boolean
```

Comment:
I know I said the default should be false, but I was mistaken. It should be true. Most amounts (such as POLYX amounts) have decimals, so divisibility is the sensible default